.gitignore

```diff
@@ -2,6 +2,18 @@
 /release
 /.vscode
 
+task_force_radio_pipe.ilk
```

Comment:
Don't change stuff that doesn't have anything to do with what this PR is doing.
You are changing the script for radio instanciation.. Why are you changing dll's in .gitignore? That has no reason to be in this PR.